src/plugins/legacy/medAlgorithmPaint/medAlgorithmPaintToolBox.cpp

```diff
@@ -1828,8 +1835,13 @@ void AlgorithmPaintToolBox::clearMask()
         m_redoStacks->remove(currentView);
     }
 
-    showButtons(false);
     resetToolbox();
+
+    if (m_addButton->isVisible() & m_eraseButton->isVisible())
```

Comment:
`&&` should be used here (even if bitwise AND works on bool type it should be avoided).
Also, this code disables the widgets only if both of them are visible. Is this the desired behaviour?

Reply:
Done for "&&" indeed. 
These 2 widgets are hidden by default in Paint toolbox. They can be displayed for instance from pipelines. So, by default in the toolbox it's best to avoid manipulating these widgets if not needed i think

Reply:
yes but hiding them **only** if they're **both** visible (i.e. not if only one is visible) is strange no?

Reply:
They are usually a block with both actions, but indeed we never know, it could be interesting to separate them. It's done